libs/s3/s3_versioning_common_test_lib.py

```diff
@@ -421,12 +424,13 @@ def upload_version(s3_test_obj: S3TestLib, bucket_name: str, object_name: str,
     versions_dict[object_name]["is_latest"] = version_id
 
 
-def delete_version(s3_ver_test_obj: S3VersioningTestLib, bucket_name: str, object_name: str,
-                   versions_dict: dict, version_id: str = None,
+def delete_version(s3_test_obj: S3TestLib, s3_ver_test_obj: S3VersioningTestLib, bucket_name: str,
+                   object_name: str, versions_dict: dict, version_id: str = None,
```

Comment:
If this is versioning specific method then version_id should be compulsory parameter.